SPECS/python-typing-extensions/python-typing-extensions.spec

```diff
@@ -0,0 +1,138 @@
+%global _description %{expand:
+The `typing_extensions` module serves two related purposes:
+- Enable use of new type system features on older Python versions. For example,
+  `typing.TypeGuard` is new in Python 3.10, but `typing_extensions` allows
+  users on Python 3.6 through 3.9 to use it too.
+- Enable experimentation with new type system PEPs before they are accepted and
+  added to the `typing` module.
+New features may be added to `typing_extensions` as soon as they are specified
+in a PEP that has been added to the [python/peps](https://github.com/python/peps)
+repository. If the PEP is accepted, the feature will then be added to `typing`
+for the next CPython release. No typing PEP has been rejected so far, so we
+haven't yet figured out how to deal with that possibility.
+Starting with version 4.0.0, `typing_extensions` uses
+[Semantic Versioning](https://semver.org/). The
+major version is incremented for all backwards-incompatible changes.
+Therefore, it's safe to depend
+on `typing_extensions` like this: `typing_extensions >=x.y, <(x+1)`,
+where `x.y` is the first version that includes all features you need.
+`typing_extensions` supports Python versions 3.7 and higher. In the future,
+support for older Python versions will be dropped some time after that version
+reaches end of life.}
+
+Summary:        Python Typing Extensions
+Name:           python-typing-extensions
+Version:        4.2.0
+Release:        6%{?dist}
+License:        PSF-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://pypi.org/project/typing-extensions/
+Source0:        %{pypi_source typing_extensions}#/typing-extensions-%{version}.tar.gz
+BuildRequires:  python3-devel
+BuildRequires:  python3-test
+BuildRequires:  python3-packaging
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-flit-core
+BuildRequires:  python3-pip
+
+BuildArch:      noarch
+
+%description %{_description}
+
+%package -n python3-typing-extensions
+Summary:        %{summary}
+
+%description -n python3-typing-extensions %{_description}
+
+%prep
+%autosetup -n typing_extensions-%{version}
+
+%generate_buildrequires
+%pyproject_buildrequires
+
+%build
+%pyproject_wheel
+
+%install
+%pyproject_install
+%pyproject_save_files typing_extensions
+
+%check
+pip3 install pytest
```

Comment:
Please set to a specific version of `pytest`, for which the tests were successful.